stratum/hal/bin/tdi/es2k/es2k_skip_p4.conf

```diff
@@ -0,0 +1,36 @@
+{
+    "chip_list": [
+    {
+        "id": "asic-0",
+        "chip_family": "mev",
```

Comment:
changes to support e2100 value are already merged, we can change this to e2100

Reply:
The instructions from Dan are to wait until after the merge.